src/cascadia/TerminalSettingsEditor/AddProfile.xaml

```diff
@@ -0,0 +1,83 @@
+        </ResourceDictionary>
+    </Page.Resources>
+
+    <ScrollViewer>
+        <StackPanel Style="{StaticResource SettingsStackStyle}">
+            <Button x:Name="AddNewButton"
+                    Click="AddNewClick"
+                    Style="{StaticResource AccentButtonStyle}">
+                <Button.Content>
+                    <StackPanel Orientation="Horizontal">
+                        <FontIcon FontSize="{StaticResource StandardIconSize}"
+                                  Glyph="&#xE710;" />
+                        <TextBlock x:Uid="AddProfile_AddNewButton"
+                                   Style="{StaticResource IconButtonTextBlockStyle}" />
+                    </StackPanel>
+                </Button.Content>
+            </Button>
+            <StackPanel Margin="{StaticResource StandardControlMargin}">
+                <local:SettingContainer x:Uid="AddProfile_Duplicate">
+                    <muxc:RadioButtons x:Name="Profiles"
+                                       ItemsSource="{x:Bind State.Settings.AllProfiles, Mode=OneWay}">
+                        <muxc:RadioButtons.ItemTemplate>
+                            <DataTemplate x:DataType="model:Profile">
+                                <Grid HorizontalAlignment="Stretch"
+                                      ColumnSpacing="8">
+
+                                    <Grid.ColumnDefinitions>
+                                        <!--  icon  -->
+                                        <ColumnDefinition Width="16" />
+                                        <!--  profile name  -->
+                                        <ColumnDefinition Width="Auto" />
+                                    </Grid.ColumnDefinitions>
+
+                                    <IconSourceElement Grid.Column="0"
+                                                       Width="16"
+                                                       Height="16"
+                                                       IconSource="{x:Bind Icon, Mode=OneWay, Converter={StaticResource IconSourceConverter}}" />
+
+                                    <TextBlock Grid.Column="1"
+                                               Text="{x:Bind Name}" />
+
+                                </Grid>
+                            </DataTemplate>
+                        </muxc:RadioButtons.ItemTemplate>
+                    </muxc:RadioButtons>
+                </local:SettingContainer>
+                <Button x:Name="DuplicateButton"
+                        Margin="{StaticResource StandardControlMargin}"
+                        Click="DuplicateClick"
+                        Style="{StaticResource AccentButtonStyle}">
+                    <Button.Content>
+                        <StackPanel Orientation="Horizontal">
+                            <FontIcon FontSize="{StaticResource StandardIconSize}"
+                                      Glyph="&#xE710;" />

```

Comment:
should we use a different glyph for Duplicate? Perhaps the Copy glyph E8C8? It looks like two pieces of paper.

Reply:
Sounds good!